addon/index.js

```diff
@@ -107,8 +111,7 @@ export function deprecationCollector(message, options, next) {
   let key = (options && options.id) || message;
   let matchKey = options && key === options.id ? 'matchId' : 'matchMessage';
 
-  self.deprecationWorkflow.deprecationLog.messages[key] =
-    '    { handler: "silence", ' + matchKey + ': ' + JSON.stringify(key) + ' }';
```

Comment:
This has been refactored as it was collecting deprecations but _also_ formatting them for final output. With that it was hard-coding `handler: 'silence'` in a place where we cannot customize it. But IMO it should have only one responsibility: collecting the raw minimal data, which is what it does now. Formatting happens in `flushDeprecations`.